### PR TITLE
[Request][ESS] Update ES|QL rule type metadata operator syntax 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/security-docs/issues/5150

Preview: 
- [Create a detection rule| Create an ES|QL rule](https://security-docs_bk_5167.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-esql-rule) -  Updated the METADATA syntax that was used in examples throughout this section. 
- [Create rule API](https://security-docs_bk_5167.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html) - Two ES|QL rule examples that included the `METADATA` field updated (thanks, @vitaliidm!) 

Serverless docs updated in https://github.com/elastic/staging-serverless-security-docs/pull/340

### CODE REVIEW

**Review by benironside — DISMISSED** [review]
LGTM.

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -318,23 +318,23 @@ Non-aggregating queries doesn't use `STATS...BY` functions and doesn't aggregate
 Here is an example non-aggregating query:
```
Comment (vitaliidm) [inline docs/detections/rules-ui-create.asciidoc:318]: there are also 2 `[metadata]` operators here:

https://github.com/elastic/security-docs/blob/036ee54e14fc3664e8cec3824b40f1f29e680951/docs/detections/api/rules/rules-api-create.asciidoc?plain=1#L1069

and here

https://github.com/elastic/security-docs/blob/036ee54e14fc3664e8cec3824b40f1f29e680951/docs/detections/api/rules/rules-api-create.asciidoc?plain=1#L1530

**Comment by vitaliidm** [comment]
Sorry, @benironside, I pushed changes in https://github.com/elastic/security-docs/pull/5167/commits/9ca2d588813c79a0831e54e7c5ef02e26ce4905e and it automatically dismissed your approval

**Review by vitaliidm — DISMISSED** [review]

**Comment by nastasha-solomon** [comment]
Using this PR to commit edits that @natasha-moore-elastic suggested in https://github.com/elastic/staging-serverless-security-docs/pull/340.

**Review by benironside — APPROVED** [review]
LGTM!

**Pull request merged**